__tests__/unittest/__snapshots__/extendOrdWithCustom.test.js.snap

```diff
@@ -0,0 +1,96 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`extendOrdWithCustom extendCustomORDContentIfExists should add new ord resources that are not supported by cap framework 1`] = `
```

Comment:
It's generated by Jest, there is no document explained it. From the [code](https://github.com/jestjs/jest/blob/main/packages/jest-snapshot/src/State.ts#L206), I guess they want to avoid same test name override.